ocs_ci/deployment/vmware.py

```diff
@@ -130,6 +130,29 @@ def attach_disk(self, size=100, disk_type=constants.VM_DISK_TYPE):
                     config.ENV_DATA.get("extra_disks", 1), vm, size, disk_type
                 )
 
+    def add_disks_per_node(
```

Comment:
why is this method part of the deployment class? I see it's being called from a non deployment test case. 
I suggest you move this method to another file, such as ocs_ci.ocs.platform_nodes.VMWareNodes

Reply:
`add_disks_per_node` method part of `VSPHEREBASE`
I added this method because I want to add disk to specific node.
On `attach_disk` method, we add disk to all worker nodes.
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/vmware.py#L128
@vavuthu What do you think?

Reply:
Since this is used as part of test case, I suggest to use https://github.com/red-hat-storage/ocs-ci/blob/1218b37872b8329a37d9b5522fe2723f7fcb6bdc/ocs_ci/ocs/platform_nodes.py#L366

attach_disk is just wrapper aroung https://github.com/red-hat-storage/ocs-ci/blob/a359b655fdfbffd7a0533d88474f03d56b465526/ocs_ci/utility/vsphere.py#L361 which is used LSO deployment types for vSphere

Reply:
@ebenahar @vavuthu done